crates/modelardb_server/src/storage/mod.rs

```diff
@@ -466,6 +476,23 @@ impl StorageEngine {
             .await
     }
 
+    /// Set the transfer batch size in the data transfer component to `new_value`. If the value
+    /// is changed successfully return [`Ok`], otherwise return [`ParquetError`].
+    pub(super) async fn set_transfer_batch_size_in_bytes(
+        &self,
+        new_value: usize,
+    ) -> Result<(), ParquetError> {
+        if let Some(ref mut data_transfer) =
+            *self.compressed_data_manager.data_transfer.write().await
+        {
+            data_transfer
+                .set_transfer_batch_size_in_bytes(new_value)
+                .await
+        } else {
+            Ok(())
```

Comment:
Maybe return an error stating that no data transfer component has been created instead of failing silently. If a user tries to changes the data transfer batch size on a node they probably think that a data transfer component exists.

Reply:
Fixed.